src/core/feed-core.js

```diff
@@ -328,6 +358,26 @@ function _getTeamNameSql(cityId) {
     : knex.raw(`CASE WHEN teams.city_id=? THEN teams.name ELSE cities.name END`, [cityId]).toString();
 }
 
+function _getNumberOfComments(id, parent_id, client){
```

Comment:
As mentioned before, it would be optimal to not run this SQL query for every feed item just to fetch the number of comments. If you are fetching a feed that is, lets say 1000 items long, this would mean 1000 additional SQL queries to database.

This logic should probably be refactored to be done with the initial SQL query that fetches feed items, and this function should only be used with the refreshCommentNumber function for refreshing single feed items comments.